qchem/CHANGELOG.md

```diff
@@ -8,17 +8,18 @@
   to load the dipole matrix elements in the atomic basis.
   [(#1698)](https://github.com/PennyLaneAI/pennylane/pull/1698)
 
```

Comment:
There is going to be another item here too as per https://github.com/PennyLaneAI/pennylane/pull/1858.